server/server.h

```diff
@@ -21,6 +21,7 @@
 
 // Qt
 #include <QObject>
+#include <atomic>
```

Comment:
I don't think `<atomic>` is part of Qt... can we put an empty line between them?

Reply:
> I don't think `<atomic>` is part of Qt... can we put an empty line between them?
Why, if std::atomic is used there, we do not need to guess whether atomic is included somewhere or not

Reply:
I'm simply suggesting this:
```C++
// Qt
#include <QObject>

#include <atomic>
```
So if in the future we include more of Qt or the STL, it doesn't become inconsistent.

Reply:
Done